src/Tests/Microsoft.NET.Sdk.WorkloadManifestReader.Tests/ManifestTests.cs

```diff
@@ -49,5 +50,80 @@ public void AliasedPackPath()
             buildToolsPack.Version.Should().Be("8.4.7");
             buildToolsPack.Path.Should().Be(Path.Combine(fakeRootPath, "packs", "Xamarin.Android.BuildTools.Win64Host", "8.4.7"));
         }
+
+        [Fact]
+        public void GivenMultiplePackRoots_ItUsesTheLastOneIfThePackDoesntExist()
```

Comment:
nit: you can spin up a new test class for these 4 methods. Do test set up in the constructor. And make these local variables fields. So you can share the set up code and make the assertions more readable.

Reply:
This is the test case for if there are duplicated packs in different pack roots? Will that happen in production? Should we just error when that happens? It would be something wrong in the dev environment.

Reply:
I don't think it will be something wrong in the dev environment.  When working on a workload, they may want to test on top of an SDK that has the same workload installed (just not the "stage 2" version of it).